woocommerce/woocommerce
# Add purchase task to experimental task lists

### All Submissions:






### Changes proposed in this Pull Request:

Adds the missing purchase task to the 2 experimental task lists.

Closes #33125  .

### How to test the changes in this Pull Request:

1. Proceed through OBW, and select the paid theme "Blooms" on the final step.
1. Be sure to opt into Marketing^
1. When brought to the Homescreen, notice that the "Add Blooms to my store" task appears.
1. Install WCAdmin Test Helper plugin, and use it to enable `woocommerce_tasklist_setup_experiment_1_*`
1. Refresh the Homescreen.
1. Note that the purchase task exists
2. Enable the `woocommerce_tasklist_setup_experiment_2_*` experiment
3. Refresh the homescreen.
4. Note the purchase task exists

### Other information:




### FOR PR REVIEWER ONLY:
